### ISSUE TITLE
Use chemacs to separate emacs profile

### ISSUE THREAD

**Issue by HumHongeKamyaab** [daviwil/runemacs#5]
You can use chemacs package to complelely seperate runmacs config folder from yr default emacs.d folder. That way it wont show any packages installed for default config in the runmacs.

**Comment by daviwil** [comment]
I've never used it before but I keep hearing about it, I'll try it!

**Comment by supermario9590** [comment]
> I've never used it before but I keep hearing about it, I'll try it!

You really should, it's absoloutly amazing. I used it to seperate Doom Emacs from the Emacs config we were building in the series.

**Comment by daviwil** [comment]
I've done a little research on it and it looks useful!  I'm considering using @alphapapa's https://github.com/alphapapa/emacs-sandbox.sh instead though since Chemacs requires that you set up your `.emacs.d` in a certain way for it to work correctly.

**Comment by alphapapa** [comment]
Yeah, Chemacs is very interesting, but `emacs-sandbox.sh` requires no configuration, so it's much simpler to use.  The name may be slightly misleading, since it easily acts as an Emacs configuration switcher as well as providing temporary sandboxes.
